### DISCUSSION TITLE
I would like to thank you.

### DISCUSSION THREAD

**Discussion by freedomtalesans** [BEEmod/BEE2-items#4199] [General]
If not for you we would be being hunted by Herobrine in Minecraft by now.

**Comment by vrad-exe** [comment]
???

**Comment by freedomtalesans** [comment]
In reference to the most recent MC update not having "removed Herobrine" in the changelog.

**Comment by vrad-exe** [comment]
I don't think they've put that in the changelogs for a while. Either way, how does that have anything to do with us, or BEEmod?

**Comment by freedomtalesans** [comment]
No, 1.16 is the first (non snapshot) update since the appearance of the creepypasta to not have it in the changelog.

**Comment by vrad-exe** [comment]
The exact version when they started and stopped putting "Removed Herobrine" in the changelog doesn't matter. This doesn't have anything to do with BEE, I'm locking the issue.

**Comment by FrozenDragon0** [comment]
This seriously has to be a joke....

**Comment by LautaroL20** [comment]
> This seriously has to be a joke....

One can´t even guess if he did this on purpose to cause annoyance, or if he is just a person with a mindset of a little boy who has nothing better to do other than open an useless issue because he can...
